### ISSUE TITLE
diki.pl bookmarklet - unecessary highlighting 

### ISSUE THREAD

**Issue by piotrex** [Quicksaver/FindBar-Tweak#123]
I found weird behavior with [diki.pl bookmarklet](http://translate.google.pl/translate?u=http%3A%2F%2Fwww.diki.pl%2Fo-slowniku%2Fzainstaluj-slownik%2F&hl=en&langpair=auto|en&tbb=1&ie=UTF-8). After double clicking on a word in line with highlighted phrase, the word also will be highlighted. I'm not think it is important issue but it's strange so I'm reporting this.

Steps to reproduce:
1. Go to any website (e.g. stackoverflow.com)
2. Start search for any term (e.g."question")
3. Enable diki.pl bookmarklet.
4. Double click on word located in line of a highlighted word (e.g. "answer" in first paragraph). This should cause open popup with this word in the dictionary.

Now the double-clicked word also will be highlighted, even if it isn't typed in search field in findbar.

**Comment by Quicksaver** [comment]
I don't think this is happening anymore with beta version 2b2, if you could try it out (https://addons.mozilla.org/firefox/addon/findbar-tweak/versions/) and let me know I'd appreciate it
